pkg/webhook/resourceinterpretercustomization/helper.go

```diff
@@ -0,0 +1,58 @@
+package resourceinterpretercustomization
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
+	"github.com/karmada-io/karmada/pkg/resourceinterpreter/configurableinterpreter/luavm"
+	"github.com/karmada-io/karmada/pkg/util/validation"
+)
+
+func validateCustomizationRule(oldRules, newRules *configv1alpha1.ResourceInterpreterCustomization) error {
+	if oldRules.Spec.Target.APIVersion != newRules.Spec.Target.APIVersion ||
+		oldRules.Spec.Target.Kind != newRules.Spec.Target.Kind {
+		return nil
+	}
+	for _, rule := range validation.AllResourceInterpreterCustomizationRules {
+		oldScript := rule.GetScript(oldRules)
+		newScript := rule.GetScript(newRules)
+		if oldScript != "" && newScript != "" {
+			return fmt.Errorf("conflicting with InterpreterOperation(%s) of existing ResourceInterpreterCustomization(%s)", rule.Name(), oldRules.Name)
+		}
+	}
+	return nil
+}
+
+func checkCustomizationsRule(customization *configv1alpha1.ResourceInterpreterCustomization) error {
+	ctx, cancel := context.WithTimeout(context.TODO(), time.Second)
```

Comment:
I want to reuse `luavm`,how much should `Timeout` be set?

Reply:
One second will be ok.

See https://github.com/argoproj/argo-cd/blob/a9da5d83d6a1d124b3a06e8873fdb9b1079a3653/util/lua/lua.go#L86